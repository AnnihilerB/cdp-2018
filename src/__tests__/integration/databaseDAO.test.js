const databaseDAO = require('../../databaseDAO');

test('Starting the database', () => {
  const pool = databaseDAO.pool;
  expect(pool).toMatchSnapshot();
});
