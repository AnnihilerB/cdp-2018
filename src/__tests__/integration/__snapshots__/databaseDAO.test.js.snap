// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Starting the database 1`] = `
Pool {
  "activatePool": [Function],
  "activeConnections": [Function],
  "end": [Function],
  "getConnection": [Function],
  "idleConnections": [Function],
  "query": [Function],
  "taskQueueSize": [Function],
  "totalConnections": [Function],
}
`;
